Sources/OpenAI/Azure/DefaultOpenAIAzureService.swift

```diff
@@ -58,6 +58,7 @@ final public class DefaultOpenAIAzureService: OpenAIService {
    public func startStreamedChat(parameters: ChatCompletionParameters) async throws -> AsyncThrowingStream<ChatCompletionChunkObject, Error> {
       var chatParameters = parameters
       chatParameters.stream = true
+      chatParameters.streamOptions = .init(includeUsage: true)
```

Comment:
@jamesrochabrun please let me know if you would like me to enable usage by default in this service

Reply:
Default usage seems reasonable to me, thanks!